data/Ousman37.json

```diff
@@ -1,17 +1,86 @@
 {
   "name": "Ousman Ethmane",
   "bio": "Passionate Front-End Developer | UI/UX Enthusiast | Building innovative web experiences",
+  "tags": [
+    "Open Source",
+    "HTML",
+    "CSS",
+    "JavaScript",
+    "React.js",
+    "Node.js",
+    "Git",
+    "GitHub",
+    "TypeScript"
+  ],
+  "evenTags": [
+    "Front-End Development",
+    "UI/UX Design",
+    "Responsive Web Design",
+    "Version Control",
+    "Web Development Tools",
+    "Problem Solving",
+    "Debugging",
+    "Team Collaboration"
+  ],
+  "socials": [
+    {
+      "name": "LinkedIn Follow me",
+      "url": "https://www.linkedin.com/in/ethmane-ousman-34885844/",
+      "icon": "FaLinkedin", 
+      "color": "#0077B5"
+    },
+    {
+      "name": "GitHub Follow me",
+      "url": "https://github.com/Ousman37",
+       "icon": "FaGithub",
+      "color": "#333"
+    },
+    {
+      "name": "Twitter Follow me",
+      "url": "https://twitter.com/Ethamne_dev",
+       "icon": "FaTwitter",
+      "color": "#1DA1F2"
+    }
+  ],
+  "testimonials": [
+    {
+      "author": "John Doe",
+      "position": "CEO, XYZ Company",
+      "quote": "Ousman is an incredibly talented front-end developer. His work has greatly improved our website's user experience."
+    },
+    {
+      "author": "Jane Smith",
+      "position": "UI/UX Designer, ABC Agency",
+      "quote": "Working with Ousman has been a pleasure. He has a keen eye for design and pays great attention to detail."
+    }
+  ],
```

Comment:
It doesn't seem to be the right way to add testimonials & may not be displayed on your profile. Please check out [these docs](https://www.biodrop.io/docs/how-to-guides/testimonials-json) to know more about testimonials

```suggestion
```